app.js

```diff
@@ -553,15 +553,25 @@ function Exele_TreeBuilder() {
             }
         }
 
-        var totalTime = efDurationSum(root);
 
-        var $totalTimeElement = $('.exele-total-time', symbolElement);
-        $totalTimeElement[0].innerHTML = 'Total event time: ' + root.data.durationMinutes.toFixed(2);
 
-        // Draw the treemap within the selected element using the data in `root`
-        treemapSelection
-            .datum(root)
-            .call(myTreemap);
+     
+        if (Object.keys(efDataHolder).length === 0) {
```

Comment:
Will this show "No events" if there is a template selected, but there are no events within that template? It will still show a blank treemap in that case won't it? i don't think that will be what we want, but i haven't been able to hit this case with the demo stuff yet.